src/main/java/com/solvd/navigator/Main.java

```diff
@@ -1,7 +1,23 @@
 package com.solvd.navigator;
 
+import com.solvd.navigator.model.util.UserInterface;
+
+import static com.solvd.navigator.model.util.AllPairShortestPath.INF;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        int[][] graph = {
```

Comment:
A service method could return this graph matrix, generated with the data from the database